.ado/templates/prep-and-pack-nuget.yml

```
@@ -1,11 +1,11 @@
 parameters:
   publishCommitId: '0'
   npmVersion: '0.0.1-pr'
-  # Note: NuGet pack expects platform-specific file spearators ('\' on Windows).
+  # Note: NuGet pack expects platform-specific file separators ('\' on Windows).
   nugetroot: $(System.DefaultWorkingDirectory)\ReactWindows
   desktopId: 'OfficeReact.Win32'
   microsoftRNId: 'Microsoft.ReactNative'
-  slices: '("x64.Release", "x64.Debug", "x86.Release", "x86.Debug", "ARM.Release", "ARM.Debug", "ARM64.Release", "ARM64.Debug")'
+  slices: '("x64.Release","x86.Release", "ARM.Release", "ARM64.Release")'
```

nit: please add space after comma.

---

Thanks, I'll update the original one in master as well in a separate PR.
```suggestion
  slices: '("x64.Release", "x86.Release", "ARM.Release", "ARM64.Release")'
```

---
In reply to: [516979992](https://github.com/microsoft/react-native-windows/pull/6417#discussion_r516979992) [](ancestors = 516979992)